x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -347,7 +347,7 @@ export function AlertsTableTGrid(props: AlertsTableTGridProps) {
       end: rangeTo,
       filters: [],
       indexNames: [indexName],
-      itemsPerPage: 10,
+      itemsPerPage: 50,
```

Comment:
It doesn't make much sense to me that we hard code `itemsPerpage` here but we also have this value inside the store. We could remove this prop and use `itemsPerPageStore` here https://github.com/elastic/kibana/blob/60af98a1b7cfe4fbb6574f8c65bc23964a2b6433/x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx#L289

Reply:
I've added https://github.com/elastic/kibana/pull/109139/files to address this, could you let me know if this looks okay? It meant adjusting the store defaults so that `itemsPerPageStore` produced 50. And *maybe* the `TGridStandaloneComponent` should've kept `itemsPerPage` as an optional prop, but this goes for the "expose when needed" route.